gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/DuplicateTripValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Arrays;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.DuplicateTripNotice;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsTrip;
+import org.mobilitydata.gtfsvalidator.table.GtfsTripTableContainer;
+
+@RunWith(JUnit4.class)
+public class DuplicateTripValidatorTest {
+  private final GtfsTrip trip1 =
+      new GtfsTrip.Builder().setCsvRowNumber(21).setTripId("trip1").build();
+  private final GtfsTrip trip2 =
```

Comment:
perhaps it would be more readable if you names the variable 'trip1_line39'

Reply:
updated to trip1_line21 and trip1_line39